versions.props

```diff
@@ -50,6 +51,7 @@ io.netty:* = 4.1.39.Final
 javax.annotation:javax.annotation-api = 1.3.2
 javax.validation:validation-api = 1.1.0.Final
 javax.ws.rs:javax.ws.rs-api = 2.0.1
+jakarta.ws.rs:jakarta.ws.rs-api = 2.1.6
```

Comment:
Given this, let's just see what comes out of https://github.com/palantir/atlasdb/pull/4677

Reply:
I think we can safely merge this separately from #4677 since jakarta would only appear on the classpath of the dialogue jars which are currently not being referenced inside of the repo.

Otherwise, I'm happy to start stacking the PRs that actually adopt dialogue on top of this one

Reply:
Ok, that stuff is now cleaned up. I've merge develop let's see if build is green